doc/Programming-guide/Using-Kaa-endpoint-SDKs/Java/index.md

```diff
@@ -155,7 +155,71 @@ the boolean argument specifies whether to generate public/private key pair autom
 The first argument can be either `DesktopKaaPlatformContext` for Java desktop or `AndroidKaaPlatformContext` for Android applications. 
 In this example, the second argument is the default implementation of `KaaClientStateListener` -- `SimpleKaaClientStateListener`, that is solely used to log the client state changes.
 And last one if set to **false** -- means using existing pre-generated public/private key pair, **true** -- means auto-generating new public/private key pair if absent.
+
+You can configure properties and `ExecutorContext` of `DesktopKaaPlatformContext` with help of it's constructors. `DesktopKaaPlatformContext` uses `FlexibleExecutorContext` implementation of `ExecutorContext` by default. You can change it by calling corresponding constructor with specific `ExecutorContext` implementation.
```

Comment:
Also FYI, we have a documentation guide [here|http://kaaproject.github.io/kaa/docs/v0.10.0/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/#markdown-formatting].